conda/parameterized/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "parameterized" %}
+{% set version = "0.6.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "caf58e717097735de0d7e15386a46ffa5ce25bb6a13a43716a8854a8d34841e2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
```

Comment:
Where does `lower` come from? Does that signify that when the packages are built then conda will read the `name` as lowercase?

Reply:
> Where does `lower` come from? 

It's built in to the [jinja2](http://jinja.pocoo.org/docs/2.9/templates/#lower) interpreter used to handle the [templating](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#templating-with-jinja) here.

> Does that signify that when the packages are built then conda will read the `name` as lowercase?

Yea, the `name` variable (defined above statically as `"parameterized"`) will be lowered before being used as the name of the package. Given that static name, lowering is a noop here, but this is all autogenerated by `conda skeleton`.